src/_03_IntroToStacks/_01_IntroToStack.java

```diff
@@ -7,29 +7,32 @@
 
 public class _01_IntroToStack {
 	public static void main(String[] args) {
-		//1. Create a Stack of Doubles
-		//   Don't forget to import the Stack class
-		
-		//2. Use a loop to push 100 random doubles between 0 and 100 to the Stack.
-		
-		//3. Ask the user to enter in two numbers between 0 and 100, inclusive. 
-		
-		//4. Pop all the elements off of the Stack. Every time a double is popped that is
-		//   between the two numbers entered by the user, print it to the screen.
-		
-		
-		//   EXAMPLE:
-		//   NUM 1: 65
-		//   NUM 2: 75
-		
-		//   Popping elements off stack...
-		//   Elements between 65 and 75:
-		//   66.66876846
-		//   74.51651681
-		//   70.05110654
-		//   69.21350456
-		//   71.54506465
-		//   66.47984807
-		//   74.12121224
+
+		Stack<Double> testStack = new Stack<>();
+		for (int i = 0; i < 100; i++) {
+			testStack.push(new Random().nextDouble() * 100);
+		}
+		String answer = JOptionPane.showInputDialog("Enter a number between 0 and 100 \n(you can include 0 and 100)");
+		String answer2 = JOptionPane.showInputDialog("Enter another number between 0 and 100");
+		int answerNum = Integer.parseInt(answer);
+		int answer2Num = Integer.parseInt(answer2);
+		int greaterNum;
+		int lesserNum;
+		if (answerNum > answer2Num) {
+			greaterNum = answerNum;
+			lesserNum = answer2Num;
+		} else {
+			greaterNum = answer2Num;
+			lesserNum = answerNum;
+		}
+		System.out.println("Popping elements off stack... \nElements between " + lesserNum + " and " + greaterNum);
+		double currentValue = 0;
```

Comment:
Is there a reason you declared the 'currentValue' variable here or should you move it to the for loop🌜